packages/neo4j/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@next-auth/neo4j-adapter",
+  "version": "0.1.0",
+  "description": "neo4j adapter for next-auth.",
+  "homepage": "https://next-auth.js.org",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "author": "Davey Brown",
+  "main": "dist/index.js",
+  "license": "ISC",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "oauth",
+    "neo4j"
+  ],
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test:watch": "./tests/test.sh -w",
+    "test": "yarn build && ./tests/test.sh",
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "peerDependencies": {
+    "next-auth": ">=4"
+  },
+  "devDependencies": {
+    "@types/uuid": "^8.3.0"
+  },
+  "dependencies": {
+    "neo4j-driver": "^4.3.0",
+    "uuid": "^8.3.2"
+  }
```

```suggestion
  "peerDependencies": {
    "neo4j-driver": "^4.3.0",
    "next-auth": ">=4",
  },
  "devDependencies": {
    "@types/uuid": "^8.3.0",
    "neo4j-driver": "^4.3.0",
  },
  "dependencies": {
    "uuid": "^8.3.2"
  }
```

Let the user use their own version of `neo4j-driver` by adding it to `peerDependencies`. Other than that, we only use the package for types, so it can be in `devDependencies` as well.

---

That makes sense. Fixed: https://github.com/usedandloved/nextauthjs-adapters/commit/4f6eef42c007b71ff7a08af2980473ef8b2af7c0